tools/NTP/ntp.conf

```diff
@@ -0,0 +1,42 @@
+driftfile /var/log/ntp/ntp.drift
```

Comment:
can you put comments at the top of this file to specify where this file would be placed and just a general overview of the settings and perhaps any settings that differ from default.